ci/aks/aks-cluster.lokocfg.envsubst

```diff
@@ -116,3 +116,7 @@ component "httpbin" {
 
   certmanager_cluster_issuer = "letsencrypt-staging"
 }
+
+component "linkerd" {
+  controller_replicas = 2
```

Comment:
Is 2 HA? Or does it form a quorum and requires 3?

Reply:
Anything more than 1 is HA